image/discourse_dev/Dockerfile

```diff
@@ -44,3 +40,7 @@ RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo ap
     apt update &&\
     apt install -y google-chrome-stable firefox-esr &&\
     npm install -g eslint babel-eslint
+
+# Install & Configure MailHog (https://github.com/mailhog/MailHog)
+RUN wget -nc -qO /usr/local/bin/mailhog https://github.com/mailhog/MailHog/releases/download/v1.0.1/MailHog_linux_amd64\
```

Comment:
We should be in the habit of doing a SHA check when we download stuff, just in case.